index.html

```diff
@@ -0,0 +1,128 @@
+    <section class="program">
+      <h2 class="program">Main Program</h2>
+      <hr>
+      <div class="desktop-flex-container">
+        <div class="program-item">
+          <i class="material-symbols-rounded">podium</i>
+          <span class="icon">Lecture</span>
+          <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+        </div>
+        <div class="program-item">
+          <i class="material-symbols-rounded">photo_frame</i>
+          <span class="icon">Exhibition</span>
+          <p>Appreciate various creations applying GhiE standards.</p>
+        </div>
+        <div class="program-item">
+          <i class="material-symbols-rounded">forum</i>
+          <span class="icon">Forum</span>
+          <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <div class="program-item">
+          <i class="material-symbols-rounded">home_repair_service</i>
+          <span class="icon">Workshop</span>
+          <p>Try creating your own work using open source license rather than just watching.</p>
+        </div>
+        <div class="program-item">
+          <i class="material-symbols-rounded">hub</i>
+          <span class="icon">Network</span>
+          <p>Get opportunities to network with other Engineers around the world.</p>
+        </div>
+      </div>
+      <button class="join">Join</button>
+      <a href="#" class="whole-program">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section id="speakers" class="speakers">
+      <!-- <h2>Featured Speakers</h2>
+      <hr>
+      <div class="speaker-flex" >
+
+        <div class="speaker-container">
+          <div class="speaker-image-container"><img src="images/sophia.jpg" alt=""><img class="checkered" src="checkered" alt=""></div>
+          <div class="speaker-text">
+            <h3>Harriet Oteng</h3>
+            <p class="position"><i>Lead Engineer of Quartz Dam Construction</i> </p>
+            <hr class="gray">
+            <p class="interest">Improve member benefits; monitor achievements within the council year and suggest other business friendly ways to Council to improve our collaboration with the public</p>
+          </div>
+        </div>
+
+      </div>
+
+      <button id="more" class="more">
+        <span class="more-text" >MORE</span>
+        <span class="fa fa-solid fa-chevron-down more-icon"></span>
+      </button> -->
+    </section>
+    <section class="partner">
+      <h2>Partner</h2>
+      <hr>
+      <div class="companies">
+        <div><i class="fab fa-brands fa-airbnb"></i></div>
+        <div><i class="fab fa-brands fa-google"></i></div>
+        <div><i class="fab fa-brands fa-amazon"></i></div>
+        <div><i class="fa fa-solid fa-snowflake"></i></div>
+        <div><i class="fab fa-brands fa-stripe"></i></div>
+      </div>
+    </section> 
+  </main>
+  <footer class="footer">
+    <p>&#169 2024 <br>Some Rights Reserved</p>
+  </footer>
```

Comment:
- Hey, please, the footer should contain your logo aligned to the left and text aligned to the right to match the design as much as you can. let's fix this. see the images below:


*design*
![Screenshot from 2023-03-16 19-04-43](https://user-images.githubusercontent.com/59397159/225712700-2ca03e2f-58d3-4afe-a323-d1c48e9967b7.png)

*Your design*
![Screenshot from 2023-03-16 18-59-48](https://user-images.githubusercontent.com/59397159/225712711-607e1826-2289-4a3f-bb6c-3cd8beabd0da.png)